src/core.c/Instant.pm6

```diff
@@ -3,7 +3,7 @@ my class DateTime { ... }
 my class Duration {... }
 
 my class Instant is Cool does Real {
-    has Real $.tai is default(0);
+    has Int $.tai is default(0);
```

Comment:
This changes the public API with `$my-instant.tai` now returning an Int instead of Real. I thought the plan was to add a `method tai(--> Real)`?

Reply:
`method tai(--> Rat:D)` is already there a couple lines down (hidden in the code fold assuming you're looking on GitHub).

Reply:
Oh, indeed! Then all is well